docs/user_guides/Gufi For Coders/main.tex

```diff
@@ -0,0 +1,226 @@
+\documentclass{article}
+
+% Language setting
+% Replace `english' with e.g. `spanish' to change the document language
+\usepackage[english]{babel}
+
+% Set page size and margins
+% Replace `letterpaper' with`a4paper' for UK/EU standard size
+\usepackage[letterpaper,top=2cm,bottom=2cm,left=3cm,right=3cm,marginparwidth=1.75cm]{geometry}
+
+% Useful packages
+\usepackage{amsmath}
+\usepackage{graphicx}
+\usepackage[colorlinks=true, allcolors=blue]{hyperref}
+
+\title{GUFI: A Coder's Guide}
+\author{Braeden Slade}
+
+\begin{document}
+\maketitle
+
+\section{Introduction}
```

Comment:
Can you move the sections into different files so that the docs aren't giant main.tex files?

Reply:
Modularized all of the documents